membership-attribute-service/app/components/TouchpointComponents.scala

```diff
@@ -64,6 +65,9 @@ class TouchpointComponents(stage: String)(implicit system: ActorSystem) {
   )
   lazy val contactRepo = new SimpleContactRepository(tpConfig.salesforce, system.scheduler, Config.applicationName)
   lazy val attrService: AttributeService = new ScanamoAttributeService(new AmazonDynamoDBAsyncClient(com.gu.aws.CredentialsProvider).withRegion(Regions.EU_WEST_1), dynamoAttributesTable)
+  lazy val zuoraAttrService: ZuoraAttributeService = new ZuoraAttributeService(accountId => zuoraRestService.getAccounts(accountId),
+    accountId => reads => subService.subscriptionsForAccountId[AnyPlan](accountId)(reads),
+    attrService.get)
```

Comment:
maybe we could use named params here to make it clearer

Reply:
Good idea. I will do this.